recipes/recipe-7-changing-default-binding-names.adoc

```diff
@@ -0,0 +1,37 @@
+# How do I override the default binding names in Spring Cloud Stream?
+
+## Problem Statement
+
+Spring Cloud Stream creates default bindings based on the function definition and signature, but how do I override these to more domain friendly names?
+
+## Solution
+
+Assume that following is your function signature.
+
+```
+@Bean
+public Function<KStream<String, Order>, KTable<String, Account>, KStream<String, EnrichedOrder>> processOrder() {
```

Comment:
Personally the issue is generic and applies to any type of function, so if I am not a Kafka user, the KStream could send me off track even though it is just a function signature that is absolutely irrelevant here. So, perhaps `Function<String, String> uppercase()`?